examples/assets_dbt_python/assets_dbt_python/repository.py

```diff
@@ -1,9 +1,8 @@
 import os
 
-import dbt_python_assets.forecasting as forecasting
-import dbt_python_assets.raw_data as raw_data
+from assets_dbt_python.assets import forecasting, raw_data
```

Comment:
I'm having a hard time keeping track of the moving pieces in this diff, but I can't seem to find any assets defined in the forecasting package

Reply:
it's in the __init__.py - btw using the github branch folder structure view might be the easiest way to review: https://github.com/dagster-io/dagster/blob/yuhan/07-31-example_overhaul_dbt_python_assets_to_recommended_project_layout/examples/assets_dbt_python/assets_dbt_python/assets/forecasting/__init__.py

the reason is we are doing load from modules, so forecasting and raw_data need to a "module" and i dont feel it's necessary to include another file in the module folder. happy to change if that's super confusing.

Reply:
ok cool! github shows a completely empty __init__.py for me, which was throwing me off